terraform/shared/scripts/setup.sh

```diff
@@ -19,7 +19,7 @@ VAULTDOWNLOAD=https://releases.hashicorp.com/vault/${VAULTVERSION}/vault_${VAULT
 VAULTCONFIGDIR=/etc/vault.d
 VAULTDIR=/opt/vault
 
-NOMADVERSION=0.8.7
+NOMADVERSION=0.9.0-beta3
```

Comment:
probably wanna go to GA now?

Reply:
thanks, i forgot about this, was overriding it externally